pkg/gce-pd-csi-driver/controller.go

```diff
@@ -106,6 +106,14 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 		case common.ParameterKeyDiskEncryptionKmsKey:
 			// Resource names (e.g. "keyRings", "cryptoKeys", etc.) are case sensitive, so do not change case
 			diskEncryptionKmsKey = v
+		case common.ParameterKeyFsType:
+			// We have to support seeing the "fstype" parameter to support
+			// migrated volumes with legacy storage classes that contain
+			// "fstype". "fstype" through volume parameters will not be
```

Comment:
Should we be translating the storageclass "fstype" parameter to our prefixed one?

Reply:
(just like we should be translating our zone/zones storageclass parameters?)

Reply:
oh you're right. The translation happens before this